server/database/seed.js

```diff
@@ -8,7 +9,15 @@ const userdescriptions = ['Globetrotter', 'Avid Traveller', 'Novice Nomad'];
 
 const userages = [1, 2, 3, 4];
 
-const seedDB = () => {};
+const seedDB = () => {
+  seedHostels(() => {
+    seedUsers(() => {
+      seedReviews(() => {
+        console.log('DB seeded');
+      })
+    })
+  })
+};
 
 const seedUsers = (callback) => {
```

Comment:
Variable names like "ageIdx" could be more descriptive to point to what they are labeling. I think "ageIndex" would be a little easier to read"